internal/cmd/kafka/command_consumergroup_describe_onprem.go

```diff
@@ -0,0 +1,67 @@
+package kafka
+
+import (
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/confluentinc/kafka-rest-sdk-go/kafkarestv3"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/kafkarest"
+	"github.com/confluentinc/cli/internal/pkg/output"
+)
+
+func (c *consumerGroupCommand) newDescribeCommandOnPrem() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "describe <consumer-group>",
+		Short: "Describe a Kafka consumer group.",
+		Args:  cobra.ExactArgs(1),
+		RunE:  c.describeOnPrem,
+	}
+
+	cmd.Flags().AddFlagSet(pcmd.OnPremKafkaRestSet())
+	pcmd.AddContextFlag(cmd, c.CLICommand)
+	pcmd.AddOutputFlag(cmd)
+
+	return cmd
+}
+
+func (c *consumerGroupCommand) describeOnPrem(cmd *cobra.Command, args []string) error {
+	restClient, restContext, err := initKafkaRest(c.AuthenticatedCLICommand, cmd)
+	if err != nil {
+		return err
+	}
+
+	clusterId, err := getClusterIdForRestRequests(restClient, restContext)
+	if err != nil {
+		return err
+	}
+
+	consumerGroup, resp, err := restClient.ConsumerGroupV3Api.GetKafkaConsumerGroup(restContext, clusterId, args[0])
+	if err != nil {
+		return kafkarest.NewError(restClient.GetConfig().BasePath, err, resp)
+	}
+
+	table := output.NewTable(cmd)
+	table.Add(&consumerGroupOut{
+		ClusterId:         consumerGroup.ClusterId,
+		ConsumerGroupId:   consumerGroup.ConsumerGroupId,
+		Coordinator:       getStringBrokerOnPrem(consumerGroup.Coordinator),
+		IsSimple:          consumerGroup.IsSimple,
+		PartitionAssignor: consumerGroup.PartitionAssignor,
+		State:             consumerGroup.State,
+	})
+	return table.Print()
```

Comment:
Since this code is the same as the cloud version, can we put this logic in a shared function?

Reply:
We'll be able to do that after we update the on-premises `kafka-rest-sdk-go` to have `Get()` functions. But currently, it doesn't support them :(